tamr_toolbox/sysadmin/instance.py

```diff
@@ -385,3 +387,213 @@ def restart_tamr(
         impersonation_password=impersonation_password,
         verbose=verbose,
     )
+
+
+def get_configs(
+    *,
+    config_names: Optional[List[str]] = None,
+    config_search_regex: Optional[str] = None,
+    user_defined_only: bool = False,
+    tamr_install_dir: str,
+    remote_client: Optional["paramiko.SSHClient"] = None,
+    impersonation_username: Optional[str] = None,
+    impersonation_password: Optional[str] = None,
+) -> Dict[str, any]:
+    """Retrieves configuration values from a Tamr instance.
+
+    Runs in a remote environment if an ssh client is specified otherwise runs in the local shell.
+    If an impersonation_username is provided, the command is run as the provided user.
+    If an impersonation_password is provided, password authentication is used for impersonation,
+    otherwise sudo is used.
+
+    Args:
+        config_names: A list of configuration names to fetch the value for,
+            when None all configurations will be fetched
+        config_search_regex: A regular expression used to filter the names of the
+            configurations to return
+        user_defined_only: Whether to filter to only user defined config
+        tamr_install_dir: Full path to directory where Tamr is installed
+        remote_client: An ssh client providing a remote connection
+        impersonation_username: A bash user to run the command as,
+            this should be the tamr install user
+        impersonation_password: The password for the impersonation_username
+
+    Returns:
+        A dictionary of Tamr configuration variables and their values
+    """
+
+    LOGGER.info(f"Retrieving {'user defined ' if user_defined_only else ''}config from Tamr.")
+
+    user_defined_flag = " --userDefined" if user_defined_only else ""
+    command = f"{tamr_install_dir}/tamr/utils/unify-admin.sh config:get{user_defined_flag}"
+
+    if config_names is not None:
+        command = f"{command} {' '.join(config_names)}"
+
+    exit_code, stdout, stderr = _run_command(
+        command=command,
+        remote_client=remote_client,
+        impersonation_username=impersonation_username,
+        impersonation_password=impersonation_password,
+        enforce_success=True,
+    )
+
+    # Tamr returns configs in yaml form, here we convert to a dictionary representation
+    configs = yaml.load(stdout, Loader=yaml.SafeLoader)
+
+    if config_search_regex is not None:
+        LOGGER.info(f"Filtering to config names containing the regex '{config_search_regex}'.")
+        compiled_regex = re.compile(config_search_regex, re.IGNORECASE)
+        return {key: value for key, value in configs.items() if compiled_regex.search(key)}
+    else:
+        return configs
+
+
+def get_config(
+    *,
+    config_name: str,
+    tamr_install_dir: str,
+    remote_client: Optional["paramiko.SSHClient"] = None,
+    impersonation_username: Optional[str] = None,
+    impersonation_password: Optional[str] = None,
+) -> any:
```

Comment:
Can you use the `Any` type from the `typing` module?
```suggestion
) -> Any:
```